contentpal/src/main/java/org/dmfs/android/contentpal/predicates/LikeArg.java

```diff
@@ -42,16 +44,16 @@ public LikeArg(@NonNull String columnName, @NonNull Object argument)
 
     @NonNull
     @Override
-    public CharSequence selection()
+    public CharSequence selection(@NonNull TransactionContext transactionContext)
     {
         return new StringBuilder(mColumnName).append(" like ?");
     }
 
 
     @NonNull
     @Override
-    public Iterable<String> arguments()
+    public Iterable<Argument> arguments(@NonNull TransactionContext transactionContext)
     {
-        return new SingletonIterable<>(mArgument.toString());
+        return new SingletonIterable<>((Argument) new ValueArgument(mArgument));
```

Comment:
Here, too, generic parameter could be used instead of the cast.

Reply:
fixed